src/py/flwr/server/superlink/state/state.py

```diff
@@ -135,6 +135,10 @@ def delete_tasks(self, task_ids: Set[UUID]) -> None:
     def create_node(self, ping_interval: float) -> int:
         """Create, store in state, and return `node_id`."""
 
+    @abc.abstractmethod
+    def restore_node(self, node_id: int, ping_interval: float) -> bool:
+        """Restore `node_id` and return True if succeed."""
```

Comment:
How is this method different from `acknowledge_ping`?

Reply:
It's the negation of acknowledge_ping